package.json

```
@@ -74,6 +100,7 @@
 		"eclint:windows": "eclint check *.js",
 		"prelint": "npm-run-posix-or-windows eclint",
 		"lint": "eslint --ext .js,.cjs,.mjs . bin/*",
+		"postlint": "tsc -P . && attw -P",
```

I'm under the impression `-P` (`--project`) defaults to the current directory, `.`. Does adding it in manually do anything you like?

```suggestion
		"postlint": "tsc && attw -P",
```

---

hm, i'm not sure - i think that in the absence of a tsconfig file, it does something better? i just got into the habit of doing this everywhere.